src/visual.ts

```diff
@@ -176,13 +181,14 @@ module powerbi.extensibility.visual {
                     objectEnumeration.push({
                         objectName: objectName,
                         properties: {
-                            show: this.settings_forecastPlot_params.show,
+                           // show: this.settings_forecastPlot_params.show,
                             forecastLength: Math.round(inMinMax(this.settings_forecastPlot_params.forecastLength,1,1000000)),
+            //                forecastDate: this.settings_forecastPlot_params.forecastDate,
```

Comment:
why all these commented lines? either delete or uncomment - don't leave them like this